src/controllers/account.js

```diff
@@ -111,6 +111,26 @@ class AccountController {
         }
     }
 
+    static async cancelEmailChanging(req, res) {
+        try {
+            const user = await UserService.findById(req.currentUser.id);
+
+            if (!user) {
+                return res.status(404).json({
+                    errors: [{ user: 'not found' }]
+                });
+            }
+            (user.newEmail = null), (user.emailConfirmToken = null);
```

Comment:
Why you do it like this?
user.newEmail = null; 
user.emailConfirmToken = null;

Reply:
These are edits  prettier,already fix